CondFormats/PPSObjects/interface/TotemT2FramePosition.h

```diff
@@ -0,0 +1,117 @@
+/****************************************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com) 
+*
+****************************************************************************/
+
+#ifndef CondFormats_PPSObjects_TotemT2FramePosition
+#define CondFormats_PPSObjects_TotemT2FramePosition
+#include "CondFormats/PPSObjects/interface/TotemFramePosition.h"
+
+#include <iostream>
+#include <string>
+
+/**
+ * Uniquely identifies the DAQ channel through which a VFAT frame has been received.
+ * 
+ * The internal representation has the following structure:
+ * \verbatim
+ * |                   32 bits raw position                    |
+ * | 12 bits | 2 bits     |  10 bits | 4 bits |       4 bits       |
+ * |  empty  | T2 payload |  FED ID  | GOH ID | index within fiber |
+ * \endverbatim
+ *
+ **/
+class TotemT2FramePosition: public TotemFramePosition {
+public:
+//  static const unsigned int offsetIdxInFiber = 0, maskIdxInFiber = 0xF;
+//  static const unsigned int offsetGOHId = 4, maskGOHId = 0xF;
+//  static const unsigned int offsetFEDId = 8, maskFEDId = 0x3FF;
+
+//  static const unsigned int offsetOptoRxId = 8, maskOptoRxId = 0x3;
+//  static const unsigned int offsetTOTFEDId = 10, maskTOTFEDId = 0x1F;
+//  static const unsigned int offsetSubSystemId = 15, maskSubSystemId = 0x7;
```

Comment:
```suggestion
```

during official review you will be asked anyway to remove all commented out lines